src/avatar.tsx

```diff
@@ -34,6 +34,9 @@ type AvatarProps = {
     badge?: boolean | number;
     'aria-label'?: string;
     dataAttributes?: DataAttributes;
+    borderAvatar?: boolean;
+    borderColor: 'border' | 'borderLow' | 'borderHigh' | 'borderSelected';
+    borderSize?: number;
```

Comment:
specs say that the border width is 1px. I prefer to limit this value to the moment. If in the future is need it, we could evolve the component to cover this need. Making the possibility open can cause unnecessary inconsistencies through different products in an ecosystem

In the future, we can cover this need through tokens if any brand is required